### PR TITLE
Infa hack2018

### PR DESCRIPTION
**Pull request by lreilly-infa**
Location capabilities

Implemented:
* Auto-location - updated API seems accurate to the building we're in.
* Location checking - unspecific. If any part of the address matches, it's valid. Invalid addresses included typos in state, but not in street name. Messages are in logger, not in UI.
* Coordinates vs Addresses - currently cannot be mixed. If they are in the same survey, even in separate questions, they will be grouped together and sent as one API request, which will not work.
* Coordinates - API requires latitude be sent in first.
* UI updates - can add location questions with multiple fields to forms, or elect to get automatic locations.
* Data storage - currently converting zip codes/addresses/etc to Long/Lat for testing. Not implemented as JSON object.

### CODE REVIEW

File: imports/ui/components/surveyForm/Question.js [hunk 1]
```diff
@@ -157,17 +198,18 @@ export default class Question extends Item {
             disabled={this.isRefused() || disabled}
             checked={!this.isRefused() && v.value == value} // eslint-disable-line
             onChange={this.handleChange}
-          /> {v.label}
+          />{' '}
```
Comment (art4ride) [inline imports/ui/components/surveyForm/Question.js]: feel free to use non breaking space `&nbsp;` in such cases

File: imports/utils/location.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  const latLngStr = latLng.join(',');
+  let address = '';
+  makeGeocodeAPICall(latLngStr)
+    .then(result => {
+      if (result.error) {
+        logger.error(escapeString(response.error.description));
+      }
+
+      if (result.standard) {
+        const stnumber = result.standard.stnumber ? result.standard.stnumber : '';
```
Comment (art4ride) [inline imports/utils/location.js]: for those cases you can also write:
`const stnumber = result.standard.stnumber || '';`

File: imports/ui/components/surveyForm/Question.js (hunk 1 above)
Comment (lreilly-infa) [reply imports/ui/components/surveyForm/Question.js]: I have used '\xa0' as '&nbsp;' was not working

File: imports/utils/location.js (hunk 2 above)
Comment (lreilly-infa) [reply imports/utils/location.js]: This code was removed when we changed APIs.

**Pull request merged**